packages/database/data-migrations/20240612115151_adds_product_config/data-migration.ts

```diff
@@ -0,0 +1,100 @@
+import { PrismaClient, SurveyType } from "@prisma/client";
+
+const prisma = new PrismaClient();
+
+type channelType = Exclude<SurveyType, "web"> | null;
+
+const main = async () => {
+  await prisma.$transaction(
+    async (tx) => {
+      const startTime = Date.now();
+      console.log("Starting data migration...");
+
+      // Fetch all products
+      const products = await tx.product.findMany({
```

Comment:
This will be a pretty big query on Formbricks Cloud. Not sure if this will work but we can try.

Reply:
since we are querying the selective data only, i think it should work, if it doesn't we can fetch data in batches and perform the operation.